# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../dhcp/dhclient-script-no-domain.patch
# Copyright (C) 2010 The OpenSDE Project
# Copyright (C) 2004 - 2006 The T2 SDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---
diff -r -u dhcp-3.0.2rc3/client/scripts/linux dhcp-3.0.2rc3-new/client/scripts/linux
--- dhcp-3.0.2rc3/client/scripts/linux	2002-11-15 02:09:09.000000000 +0100
+++ dhcp-3.0.2rc3-new/client/scripts/linux	2005-01-16 00:13:49.664021176 +0100
@@ -33,7 +35,7 @@
 }
 
 # Must be used on exit.   Invokes the local dhcp client exit hooks, if any.
-exit_with_hooks() {
+function exit_with_hooks() {
   exit_status=$1
   if [ -f /etc/dhclient-exit-hooks ]; then
     . /etc/dhclient-exit-hooks
